apps/costcontrol/test/unit/startup_test.js

```diff
@@ -427,4 +427,73 @@ suite('Application Startup Modes Test Suite >', function() {
       CostControlApp.init();
     }
   );
+
+  test(
+    'DSDS Ensure the FTE will be closed the second time when there are a data' +
+      ' slot change',
+    function(done) {
+      var defaultDataSlotId = 0, newDataSlotId = 1,
+          dataSlot = 'ril.data.defaultServiceId';
+      var applicationMode = 'DATA_USAGE_ONLY';
+      function setupSims() {
+        MockNavigatorSettings.mSettings[dataSlot] = defaultDataSlotId;
+
+        setupCardState({cardState: 'ready'});
+        window.Common = new MockCommon({ });
+        window.ConfigManager = new MockConfigManager({
+          fakeSettings: { fte: true },
+          applicationMode: applicationMode
+        });
+
+        MockNavigatorMozMobileConnections.mAddMobileConnection(
+          {  iccId: '0000000' } ,1);
+        MockNavigatorMozIccManager.addIcc('0000000', {});
+
+        SimManager.reset();
+      }
+
+      function _onftestarted(evt) {
+        // Simulating the last step of the fte, the start app is launched.
```

Comment:
Instead of explaining this in a commentary, why to not change the name of the function to be more meaningful?
For instance, how about `_startApp`.